File: src/javas/proglem/selection_Statements_Programs/Largest_Twree_Numbers_P_02.java
package javas.proglem.selection_Statements_Programs;

//	Find the largest of three numbers
public class Largest_Twree_Numbers_P_02 {
    public static void main(String[] args) {

        int a =30,b=20,c=40;

        if (a>b && a>c){
            System.out.println("a is greater");
        }else if(b>a && b>c){
            System.out.println("b is greater");
        }  else{
            System.out.println("c is greater");
        }

    //OR
      /*  if (a>b){
            if (a>c){
                System.out.println("a is greater");
            }else {
                System.out.println("c is greater");
            }
        } else if (b>c) {
            System.out.println("b is greater");

        }else {
            System.out.println("c is greater");
        }*/
    }
}

File: src/Kotlin/problems/Selection Statements Programs/Largest_Twree_Numbers_P_02.kt
package Kotlin.problems.`Selection Statements Programs`
//I.	Find the largest of three numbers
fun main() {
    val a = 30
    val b = 20
    val c = 40

    if (a > b && a > c) {
        println("a is greater")
    } else if (b > a && b > c) {
        println("b is greater")
    } else {
        println("c is greater")
    }
}